docs/design/mockups/phase-4-homepage/round-r10-card-structure-fix.html

```
@@ -0,0 +1,384 @@
+    </style>
+  </head>
+  <body>
+    <header class="page">
+      <div class="meta">Phase 4.5 · R10 · NewsCard structure fix</div>
+      <h1>Flush-edge image vs nested frame.</h1>
+      <p>
+        Comparing what we shipped (left) against what your source design
+        shows (right). The original news cards use a SINGLE outer frame
+        with the image flush to the top + side edges, divided from the
+        meta panel by an ink rule. My mockups have been showing a nested
+        `<TapedFigure>` inside the outer `<TapedCard>` — two frames with
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </tapedcard></tapedfigure> ], start tag match failed [ <TapedCard> ] on line 287.

(tag-pair)